Ginger/GingerCoreCommon/GeneralLib/JSONLib/JsonExtended.cs

```diff
@@ -137,6 +137,22 @@ public JToken GetToken()
             return JT;
         }
 
+        //
+        // Summary:
+        //     Selects a Newtonsoft.Json.Linq.JToken using a JPath expression. Selects the token
+        //     that matches the object path.
+        //
+        // Parameters:
+        //   path:
+        //     A System.String that contains a JPath expression.
+        //
+        // Returns:
+        //     A Newtonsoft.Json.Linq.JToken, or null.
+        public JToken SelectToken(string path)
+        {
+            return JT.SelectToken(path);
+        }
+
 
```

Comment:
please add the Function descripption as required using /// (3)

Reply:
Didn't understand